tests/cypress/tests/e2e/mcpar/dashboard.feature

```diff
@@ -0,0 +1,91 @@
+                | Test Program 2 | 01012022   | 12212024   | false        |
+
+        Scenario Outline: State users can Edit programs
+            Given I click the "Edit Program" button
+            And these form elements are prefilled:
+                | programName              | text           | Test Program 2 |
+                | reportingPeriodStartDate | text           | 01/01/2022     |
+                | reportingPeriodEndDate   | text           | 12/21/2024     |
+                | combinedData             | singleCheckbox | false          |
+
+            When these form elements are edited:
+                | programName              | text           | <title>        |
+                | reportingPeriodStartDate | text           | <startDate>    |
+                | reportingPeriodEndDate   | text           | <endDate>      |
+                | combinedData             | singleCheckbox | <combinedData> |
+            And I click the "Save" button
+            Then there is the active program with "<title>"
+
+            Examples: #Edited Data
+                | title          | startDate  | endDate    | combinedData |
+                | Edited Program | 06/22/2021 | 12/05/2025 | true         |
+
+
+    Rule: Admin User Logged In
+        Background: Admin User on homepage
+            Given I am logged in as an admin user
+            And I am on "/"
+
+        Scenario: Admin users cannot create programs
+            Given these form elements are filled:
+                | state | dropdown | Minnesota |
+
+            When I click the "Go to Report Dashboard" button
+            Then there is no way to create a program
+
+        Scenario: Admin users can archive programs
+            Given these form elements are filled:
+                | state | dropdown | Minnesota |
+
+            When I click the "Go to Report Dashboard" button
+            And I click the "Archive" button
+            And I click the "Archive" button
+            Then the program is archived
+
+        Scenario: Admin users can unarchive programs
+            Given these form elements are filled:
+                | state | dropdown | Minnesota |
+
+            When I click the "Go to Report Dashboard" button
+            And I click the "Unarchive" button
+            Then one program is archived and the other is unarchived
+
+        Scenario: Admin users can rearchive programs
+            Given these form elements are filled:
+                | state | dropdown | Minnesota |
+
+            When I click the "Go to Report Dashboard" button
+            And I click the "Archive" button
+            Then the program is archived
+
+    Rule: State User Logged In
+        Background: State User on MCPAR page
+            Given I am logged in as a state user
+            And I am on "/mcpar"
+
+        Scenario: State users can't see archived programs
+            Given I am on "/mcpar"
+            Then there are no active programs
```

Comment:
Will this consistently pass? The sidebar tests and the form tests create programs, plus we create programs, so this won't always be the case, right?

Reply:
I've gone ahead and cleaned up the sidebar test so that it archives the program it creates. The form also archives the program it creates. That should keep it clean for this test to pass. However, if we went in there and added a program and didn't delete it then yes, this would fail. That could be ok, but I could also go a step further and make sure there are no active programs by the name of Test Program or Test Program 2 (The 2 created in the form test) to make this slightly more robust. Happy to do either or